src/main/java/com/abcdedu_backend/member/service/AdminMemberService.java

```diff
@@ -26,4 +33,22 @@ public Page<AdminSearchMemberResponse> searchMembers(Pageable pageable, MemberSe
         log.info("필터에 맞는 member 조회 성공");
         return members.map(AdminSearchMemberResponse::fromMember);
     }
+
+    @Transactional
+    public void updateMembersRole(MemberRole roleName, List<ChangeMemberRoleRequest> requests) {
+        checkIsAllowedRoleChange(roleName); // to관리자 변경 방지
+
+        List<Long> ids = requests.stream()
+                .map(ChangeMemberRoleRequest::memberId)
+                .toList();
+        List<Member> members = memberRepository.findAllByIdIn(ids);
+        members.forEach(member -> member.updateRole(roleName));
+        memberRepository.saveAll(members);
+    }
```

Comment:
p3: 이부분 saveAll(members) 없어도 다 수정 되지 않나요? findall에서 영속상태인것 같아서요
```suggestion
        members.forEach(member -> member.updateRole(roleName));
    }
```

Reply:
아 맞습니다! 수정했어요! 

[cadeb5b](https://github.com/team-abcdedu/abcdedu-backend/pull/159/commits/cadeb5ba6573f51dea3d9c588ca3ebe81eac512c)

Reply:
리뷰 감사합니다요